migrations/1683909884000_inscriptions_per_block.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export function up(pgm: MigrationBuilder): void {
+  pgm.createTable('inscriptions_per_block', {
+    block_height: {
+      type: 'int',
```

Comment:
Use `bigint` for all fields

Reply:
Ah, didn't see the expand-int-size migration ✅